src/nih_wayfinding/app/scripts/routing/abbreviate-filter.js

```diff
@@ -0,0 +1,105 @@
+
+(function () {
+    'use strict';
+
+    var directions = {
+        north: 'N',
+        northeast: 'NE',
+        east: 'E',
+        southeast: 'SE',
+        south: 'S',
+        southwest: 'SW',
+        west: 'W',
+        northwest: 'NW',
+        n: 'N',
```

Comment:
Leaving as is, since if I remove these entries, they are not always cleaned to caps case. As is, something like 's Baltimore ave' -> 'S Baltimore Ave' which is helpful if not terribly common.